src/blocks.tsx

```diff
@@ -182,8 +182,15 @@ export const InformationBlock: React.FC<InformationBlockProps> = ({
 interface HighlightedValueBlockProps {
     tag?: RendersNullableElement<typeof Tag>;
 
-    value: string | Array<string>;
-    text: string | Array<string>;
+    mainHeading: {
```

Comment:
as a suggestion, you could use string | {value: string, text: string}, so if you pass a string here, it uses it as 'value'. This will reduce verbosity in places in which you don't need a 'text'

Reply:
Thank you for the suggestion, I tried to use this way but ended up giving some errors